### PR TITLE
MAINT - Removed TODOs

### PR DESCRIPTION
**Pull request by blen-desta**
- Removed TODOs that are not relevant
- Removed TODOs in exception codes (since no one is looking at the existing one and we have tickets tracking them)
- Created cards for remaining TODOs

### CODE REVIEW

File: ctk/common/src/main/kotlin/org/codice/compliance/utils/TestCommon.kt [hunk 1]
```diff
@@ -120,13 +120,6 @@ class TestCommon {
          */
         fun parseErrorResponse(response: Response): IdpResponseDecorator {
             return if (response.header("Location") != null) {
-                /*
-                 * TODO "Manually change DDF IdP to respond with 302/303 status code for Redirect"
```
Comment (bakejeyner) [inline ctk/common/src/main/kotlin/org/codice/compliance/utils/TestCommon.kt]: ❓ This TODO was to remind us that when DDF is fixed to return the correct HTTP status code. What was the reasoning behind taking it out?

File: ctk/common/src/main/kotlin/org/codice/compliance/verification/binding/RedirectBindingVerifier.kt [hunk 2]
```diff
@@ -93,8 +93,6 @@ class RedirectBindingVerifier(private val response: IdpRedirectResponseDecorator
      * 3.4.6 Error Reporting
      */
     private fun verifyHttpRedirectStatusCode() {
-        // TODO remove the 200 check when "Manually change DDF IdP to respond with 302/303 status
```
Comment (bakejeyner) [inline ctk/common/src/main/kotlin/org/codice/compliance/verification/binding/RedirectBindingVerifier.kt]: ❓ See above question.

File: ctk/common/src/main/kotlin/org/codice/compliance/verification/binding/RedirectBindingVerifier.kt [hunk 3]
```diff
@@ -114,8 +112,6 @@ class RedirectBindingVerifier(private val response: IdpRedirectResponseDecorator
      * 3.4.6 Error Reporting
      */
     private fun verifyHttpRedirectStatusCodeErrorResponse() {
-        // TODO remove the 200 check when "Manually change DDF IdP to respond with 302/303 status
```
Comment (bakejeyner) [inline ctk/common/src/main/kotlin/org/codice/compliance/verification/binding/RedirectBindingVerifier.kt]: ❓ See above question.

File: ctk/common/src/main/kotlin/org/codice/compliance/utils/TestCommon.kt (hunk 1 above)
Comment (blen-desta) [reply ctk/common/src/main/kotlin/org/codice/compliance/utils/TestCommon.kt]: I removed it cause we aren't doing that anymore. We will be responding exclusively with POST. We still might want to do it for log out so I will document that in the ticket.

File: ctk/common/src/main/kotlin/org/codice/compliance/verification/binding/RedirectBindingVerifier.kt (hunk 2 above)
Comment (bakejeyner) [reply ctk/common/src/main/kotlin/org/codice/compliance/verification/binding/RedirectBindingVerifier.kt]: ❗️ Either leave a comment about allowing the 200 response code, or take out the extra check (`response.httpStatusCode != HttpStatusCodes.STATUS_CODE_OK`)

File: ctk/common/src/main/kotlin/org/codice/compliance/verification/binding/RedirectBindingVerifier.kt (hunk 3 above)
Comment (bakejeyner) [reply ctk/common/src/main/kotlin/org/codice/compliance/verification/binding/RedirectBindingVerifier.kt]: ❗️ See above exclamation.

**Pull request merged**